### DISCUSSION TITLE
Help in accessing Azure pack from GitHub Education

### DISCUSSION THREAD

**Discussion by S11196050** [community/community#140362] [GitHub Education]
### Hi everyone,

I have gotten GitHub education. I want to use the Azure services provided but I cannot access it. Can anyone help me to redeem or access it?

**Comment by davevad93** [comment]
Hi @S11196050 ,

This is a known issue, #131336, #135739, to solve it you can do the following:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

The users in the discussions I provided above solved it by doing that.
